config/cloud.cfg.tmpl

```diff
@@ -90,11 +120,15 @@ cloud_config_modules:
  - emit_upstart
  - snap
 {% endif %}
+{% if variant not in ["photon"] %}
  - ssh-import-id
  - locale
+{% endif %}
  - set-passwords
-{% if variant in ["rhel", "fedora"] %}
+{% if variant in ["rhel", "fedora", "photon"] %}
+{% if variant not in ["photon"] %}
```

Comment:
Do you want spacewalk or no? Only one of these lines should be used.

Reply:
I don't want spacewalk but I want yum-add-repo.